pinot-clients/pinot-java-client/src/main/java/org/apache/pinot/client/Connection.java

```diff
@@ -59,14 +62,25 @@ public PreparedStatement prepareStatement(String statement) {
   }
 
   /**
-   * Executes a PQL statement.
+   * Executes a SQL statement.
    * @param statement The statement to execute
    * @return The result of the query
    * @throws PinotClientException If an exception occurs while processing the query
    */
   public ResultSetGroup execute(String statement)
       throws PinotClientException {
-    return execute(null, new Request("pql", statement));
+    return execute(null, new Request(SQL, statement));
```

Comment:
better to leave the existing client as it is and add a new client api to query the new end point. We dont want the existing usage to break.